### ISSUE TITLE
Release to production: end of July

### ISSUE THREAD

**Issue by ajrobbins** [project-icp/bee-pollinator-app#247]
Final release to production on this contract should occur on Wednesday 7/26 or Thursday 7/27 (fallback). Release must occur in the month of July in order to be covered in the contract.

**Comment by ajrobbins** [comment]
Issues to be included in this release are here: https://github.com/project-icp/bee-pollinator-app/milestone/7

We are still on track for release next Weds or Thursday. @hectcastro Does that work with your schedule? Which would you prefer?

**Comment by hectcastro** [comment]
Yeah, should be straightforward since we did one fairly recently. I created https://github.com/azavea/operations/issues/80 to track it on our end because we can't have multiple milestones on an issue.
